platform/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@
 # List of all optional platform libraries available.
 add_library(mbed-os-psa INTERFACE)
 
+if("EXPERIMENTAL_API" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(FEATURE_EXPERIMENTAL_API)
+endif()
```

Comment:
This needs to be deleted as the subdirectory is already included from the top level `CMakeLists.txt`.